server/servers/api/src/main/scala/com/prisma/api/server/ApiServer.scala

```diff
@@ -67,8 +67,10 @@ case class ApiServer(
   }
 
   val innerRoutes = extractRequest { _ =>
-    val requestId            = requestPrefix + ":api:" + createCuid()
-    val requestBeginningTime = System.currentTimeMillis()
+    val requestId             = requestPrefix + ":api:" + createCuid()
+    val requestBeginningTime  = System.currentTimeMillis()
+    val logSlowQueries        = EnvUtils.asBoolean("SLOW_QUERIES_LOGGING").getOrElse(false)
```

Comment:
Move the env var resolution out of the hot path please, else they are loaded on each request.

Reply:
Done.